google/cloud/bigtable/row_set.py

```diff
@@ -109,6 +109,24 @@ def add_row_range_from_keys(
         row_range = RowRange(start_key, end_key, start_inclusive, end_inclusive)
         self.row_ranges.append(row_range)
 
+    def add_row_range_by_prefix_from_keys(self, row_key_prefix):
+        """Add row range to row_ranges list that start with the row_key_prefix from the row keys
+
+        For example:
+
+        .. literalinclude:: snippets_table.py
+            :start-after: [START bigtable_row_range_by_prefix_from_keys]
+            :end-before: [END bigtable_row_range_by_prefix_from_keys]
+
+        :type row_key_prefix: str
+        :param row_key_prefix: Add all rows that start with this row key prefix.
+                            Prefix cannot be zero length."""
+
+        end_key = row_key_prefix[:-1] + chr(ord(row_key_prefix[-1]) + 1)
```

Comment:
That seems strange. If you'll pass "row" (as you did in your sample), you'll get "rox". What it should mean?
Sounds to me, in this variable we should have a name of the key on which we'd like to stop fetching data from table.

In Billy's example there is a "#" symbol. After +1 it'll become "$" which is the "end of line" regex symbol. It sound logical, but incrementing the code of the last symbol overall doesn't seem correct to me.

Reply:
`#` and `$` doesn't work here we need to remove manually , Please see the implementation of prefix in go and node:

 Following links of integration test :
Go:  https://github.com/googleapis/google-cloud-go/blob/51eb5ee54e44743973ca6a638660fd1067b17365/bigtable/integration_test.go#L673

Node:
https://github.com/googleapis/google-cloud-node/pull/1802/files#diff-5528ad90973774d4a89e43a0462bf388R233